cfme/tests/test_appliance.py

```diff
@@ -180,3 +181,26 @@ def test_keys_included(appliance, soft_assert):
         soft_assert(appliance.ssh_client.run_command(
             "test -e /var/www/miq/vmdb/certs/{}".format(k)).success,
             "{} was not included in the build".format(k))
+
+
+@pytest.mark.uncollectif(lambda appliance: appliance.version < '5.10')
+def test_codename_in_log(configured_appliance):
+    log = '/var/www/miq/vmdb/log/evm.log'
+    configured_appliance.ssh_client.run_command('echo > {}'.format(log))
+    configured_appliance.ssh_client.run_command('appliance_console_cli --server=restart')
+
+    def codename_in_log():
+        configured_appliance.ssh_client.run_command(r'egrep "Codename: \w+$" {}'.format(log))
+    wait_for(codename_in_log, timeout=30)
+
+
+@pytest.mark.uncollectif(lambda appliance: appliance.version < '5.10')
+def test_codename_in_stdout(configured_appliance):
+    cursor = configured_appliance.ssh_client.run_command(
+        'journalctl -u evmserverd --show-cursor | tail -n1').output.split('-- cursor: ')[1]
+    configured_appliance.ssh_client.run_command('appliance_console_cli --server=restart')
+
+    def codename_in_stdout():
+        configured_appliance.ssh_client.run_command(
+            r'journalctl -u evmserverd -c "{}" | egrep "Codename: \w+$"'.format(cursor))
+    wait_for(codename_in_stdout, timeout=30)
```

Comment:
Same question... 
`def codename_in_stdout()` method should return `bool`

Reply:
Thanks